bootq/src/main/java/life/genny/bootq/models/BatchLoading.java

```diff
@@ -302,12 +302,12 @@ public void persistEntities(Map<String, Map<String, String>> project, String rea
 
             try {
 				beUtils.updateBaseEntity(baseEntity, false);
-                if (count++ % LOG_BATCH_SIZE == 0)
-                    log.debugf("Saved %s baseEntitys. Continuing...", count);
 			} catch (Exception e) {
                 String entityInfo = realmName + ":" + entry.getValue().get("code");
                 loadReport.addPersistError(EReportCategoryType.BASE_ENTITY, entityInfo, e);
             }
+            if (count++ % LOG_BATCH_SIZE == 0)
```

Comment:
Moving this out for try would result in the count getting incremented even when entities are not persisted to infinispan/DB. You might want to maintain 2 separate counters, one to track total number of entities processed and the other to track the number of entities successfully saved.

Reply:
I just aimed to keep this consistent with the other methods. You want to move them all in to the try for every method?

Reply:
Yeah.. otherwise, the printed count would be misleading. Worth changing all methods to provide correct information.